code/modules/mining/mine_items.dm

```diff
@@ -166,6 +166,80 @@ proc/move_mining_shuttle()
 	light_power = 2
 	light_color = LIGHT_COLOR_BULB
 
+/***********************Wall-mounted Lanterns************************/
```

Comment:
I would rather you put these in their own file.
mine_items is one of those files that needs organisation, not bloat.